src/ResourceManager/IotHub/Commands.IotHub/Microsoft.Azure.Commands.IotHub.format.ps1xml

```diff
@@ -0,0 +1,151 @@
+﻿<?xml version="1.0" encoding="utf-8" ?>
+<Configuration>
+  <ViewDefinitions>
+    <View>
+      <Name>
+        Microsoft.Azure.Commands.Management.IotHub.Models.PSCertificate
+      </Name>
+      <ViewSelectedBy>
+        <TypeName>Microsoft.Azure.Commands.Management.IotHub.Models.PSCertificate</TypeName>
+      </ViewSelectedBy>
+      <TableControl>
+        <TableHeaders>
+          <TableColumnHeader>
```

Comment:
Shouldn't you have the other header labels?  Or does it automatically have the correct header for the first three categories?

Reply:
Automatically have the correct header